proto/reaction.proto

```diff
@@ -375,19 +452,19 @@ message TemperatureConditions {
 }
 
 message PressureConditions {
+  // TODO Consider renaming given adjustment in meaning
   message PressureControl {
     enum PressureControlType {
       UNSPECIFIED = 0;
       CUSTOM = 1;
       AMBIENT = 2;
-      BALLOON = 3;
-      SEALED = 4; // Fully sealed vessel (e.g., microwave vial).
-      SEPTUM_WITH_NEEDLE = 5;  // Slight positive pressure maintained
-      RELEASEVALVE = 6;
-      BPR = 7; // Back pressure regulator, as used in flow synthesis.
+      CONSTANT = 3;  // Slight positive pressure, e.g., with a gas line inlet.
```

Comment:
Does `CONSTANT` imply `SLIGHT_POSITIVE`? The latter might provide a clearer distinction with `PRESSURIZED`, but I feel like high pressure reactions are often also constant pressure.

Reply:
Yes, went back and forth on this. `SLIGHT_POSITIVE` is more clear. Change made.